util-linux.yaml

```diff
@@ -0,0 +1,162 @@
+package:
+  name: util-linux
+  version: 2.38.1
+  epoch: 0
+  description: Random collection of Linux utilities
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      attestation: TODO
+      license: |-
+        GPL-3.0-or-later AND GPL-2.0-or-later AND GPL-2.0-only AND PL-1.0-only AND LGPL-2.1-or-later AND LGPL-1.0-only AND BSD-1-Clause AND BSD-3-Clause AND BSD-4-Clause-UC AND MIT AND Public-Domain
```

Comment:
@rawlingsj Is the `PL-1.0-only` here meant to be `GPL-1.0-only`?  This is failing the NTIA license check as an invalid SPDX license